core/system.go

```diff
@@ -949,3 +957,40 @@ func (s *ABSystem) RemoveStageFile() error {
 	PrintVerboseInfo("ABSystem.RemoveStageFile", "stage file removed")
 	return nil
 }
+
+// isAbrootRunning checks if an instance of the `abroot` command is running
+// other than the current process
+func isAbrootRunning() bool {
+	pid := os.Getpid()
+	procs, err := os.ReadDir("/proc")
+	if err != nil {
+		return false
+	}
+
+	for _, file := range procs {
+		if file.IsDir() {
+			if _, err := strconv.Atoi(file.Name()); err == nil {
+				cmdline, err := os.ReadFile("/proc/" + file.Name() + "/cmdline")
+				if err == nil && strings.Contains(string(cmdline), "abroot") {
```

Comment:
This should also check if the `/proc/<pid>/exe` symlink points to an abroot executable, like [busybox pidof](https://github.com/brgl/busybox/blob/master/libbb/find_pid_by_name.c#L91-L94) and [procps-ng](https://gitlab.com/procps-ng/procps/-/blob/master/src/pidof.c?ref_type=heads#L199) do